extensions/google-shopping-ads/index.md

```diff
@@ -61,10 +61,14 @@ To add the Google API key:
 
     ![Add the API Key](images/config-api-key.png)
 
+    {:.bs-callout .bs-callout-info}
+You can also [Enable Service Logs](https://docs.magento.com/m2/ce/user_guide/configuration/services/channels.html) during this step. This option is disabled by default and should only be used when needed for troubleshooting, as continued logging will negatively impact performance.
```

Comment:
Suggestion: if this is strictly for version 2.0.0, may need to note that.

Reply:
added "Beginning with version 2.0.0, you...."

Reply:
this has been updated to 3.0 now.